src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class IsoscelesTrapezoid extends Figure {
+    public static final int MAXIMUM_SIZE = 50;
+    private int bottom;
+    private int top;
+    private int height;
+    private Random random = new Random();
```

Comment:
The Random object should be created in the FigureSupplier class and passed to the constructor of the figure instead of being generated inside the figure class.